go.mod

```diff
@@ -22,3 +22,5 @@ replace (
 	howett.net/plist => github.com/DHowett/go-plist v0.0.0-20181124034731-591f970eefbb
 	k8s.io/client-go => k8s.io/client-go v0.17.4 // Required by prometheus-operator
 )
+
+replace github.com/open-cluster-management/api => open-cluster-management.io/api v0.0.0-20200610161514-939cead3902c
```

Comment:
Merge this replace with above. Build failed. You may need to replace the client-go version to `v0.18.3`

Reply:
That was the API change between 0.17 and 0.18